setup.py

```diff
@@ -41,7 +39,5 @@
         ],
     },
     # *Minimum* requirements
-    install_requires=['numpy', 'netCDF4'],
-    ext_modules=cythonize(["src/cablab/*.pyx"]),
-    include_dirs=[numpy.get_include()]
+    install_requires=['numpy', 'netCDF4', 'gridtools'],
```

Comment:
It has been developed by @forman very recently: https://github.com/CAB-LAB/gridtools